### PR TITLE
Decentraland images and profile screen layout

### LINKED ISSUE #327: Make sure my profile screen content has spacing on the right
IMG_3940

### LINKED ISSUE #339: Add screenshots to Decentraland README
(Since we regularly share a directly link to this dir)

### PR DESCRIPTION
**Pull request by pcowgill**
### Issue link

https://github.com/tasitlabs/tasit/issues/339
https://github.com/tasitlabs/tasit/issues/327

### Auto-close the issue?

Closes #339 
Closes #327 

### Types of changes

Bug fix (non-breaking change that fixes an issue)

Docs (modified documentation or code comments)

Technical debt (a code change that doesn't fix a bug or add a feature but makes something clearer for devs)

### Notes

The text on the "My profile" screen is wrapping now:
MyProfileSmall

### CODE REVIEW

File: decentraland/components/presentational/MyProfileCreationStatusItem.js [hunk 1]
```diff
@@ -56,12 +57,14 @@ MyProfileCreationStatusItem.propTypes = {
...
+    flexWrap: "wrap",
     flexDirection: "row",
     backgroundColor: Colors.backgroundColor,
-    alignItems: "flex-start",
+    alignItems: "center",
     justifyContent: "flex-start",
-    marginTop: responsiveHeight(1),
-    height: responsiveHeight(6),
+    marginTop: responsiveHeight(2),
+    // height: responsiveHeight(FONT_SIZE),
```
Comment (pcowgill) [inline decentraland/components/presentational/MyProfileCreationStatusItem.js]: @marcelomorgado Happy to remove this before merging if you're happy with this change

**Comment by pcowgill** [comment]
@marcelomorgado This is how the `decentraland` dir README looks in this feature branch:

https://github.com/tasitlabs/tasit/tree/feature/decentraland-images/decentraland

**Comment by marcelomorgado** [comment]
> @marcelomorgado This is how the `decentraland` dir README looks in this feature branch:
> 
> https://github.com/tasitlabs/tasit/tree/feature/decentraland-images/decentraland

Niiiiice!!!!!

File: decentraland/components/presentational/MyProfileCreationStatusItem.js (hunk 1 above)
Comment (pcowgill) [reply decentraland/components/presentational/MyProfileCreationStatusItem.js]:
```suggestion
```

**Comment by pcowgill** [comment]
@marcelomorgado This also made me think that we might want to prioritize a landing page for this app at somet point with these screenshots at `tasit.io/decentraland` or `tasit.io/apps`

**Pull request merged**